src/Beaches.js

```diff
@@ -49,13 +50,15 @@ class Beaches extends Component {
   }
 
   filterByCounty = (e) => {
-    let filteredBeaches = this.state.beaches.filter((beach) => {
-      if(e.target.innerText === beach.county){
-        this.setState({
-          beaches: filteredBeaches
-        })
+    let beaches = this.state.beaches 
```

Comment:
This looks good. As discussed, need to add a way to allow the counties to still be accessible after the first filter or go back to all the counties.